lambdas/build-proven-user-identity-details/src/main/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/domain/ProvenUserIdentityDetails.java

```diff
@@ -7,15 +7,14 @@
 import lombok.EqualsAndHashCode;
 import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
 import uk.gov.di.ipv.core.library.domain.Address;
-import uk.gov.di.ipv.core.library.domain.BaseResponse;
 
 import java.util.List;
 
 @EqualsAndHashCode(callSuper = false)
 @ExcludeFromGeneratedCoverageReport
 @Data
 @Builder
-public class ProvenUserIdentityDetails extends BaseResponse {
```

Comment:
(`BaseResponse` looks a bit suspect in general, but we don't need it here - we can serialize this class normally)